JSDemos/menuMeta.json

```
@@ -5087,7 +5087,6 @@
           {
             "Title": "List Editing and API",
```

Why is the demo titled like this, when you say in the description that it "illustrates item deletion". Can the title be more specific them? 

```suggestion
            "Title": "Item Deletion UI",
```